ts/src/alpaca.ts

```diff
@@ -337,8 +335,7 @@ export default class alpaca extends Exchange {
         if (limit !== undefined) {
             request['limit'] = limit;
         }
-        const method = this.safeString (this.options, 'fetchTradesMethod', 'cryptoPublicGetCryptoTrades');
-        const response = await this[method] (this.extend (request, params));
+        const response = await this.cryptoPublicGetCryptoTrades (this.extend (request, params));
```

Comment:
same here

Reply:
@sc0Vu https://alpaca.markets/docs/crypto/crypto-data/
![image](https://github.com/ccxt/ccxt/assets/43336371/d76f3124-dd79-4c38-8002-b75f0a49a049)
![image](https://github.com/ccxt/ccxt/assets/43336371/89489065-330e-4c56-b142-5b47c0c56c86)

maybe we need to add them to the implicit API?

Reply:
@carlosmiei uh, they have upgraded to beta3, and added locale parameter (required) in path, eg `cryptoPublicGetCryptoLocLatestBars ()`, we probably need to upgrade as well.

https://docs.alpaca.markets/reference/cryptolatestbars

https://docs.alpaca.markets/reference/cryptolatesttrades